Sources/Preferences/ToolbarItemStyleViewController.swift

```diff
@@ -7,7 +7,7 @@ final class ToolbarItemStyleViewController: NSObject, PreferencesStyleController
 
     weak var delegate: PreferencesStyleControllerDelegate?
 
-    init(preferences: [PreferencePane], toolbar: NSToolbar, centerToolbarItems: Bool = true) {
+    init(preferences: [PreferencePane], toolbar: NSToolbar, centerToolbarItems: Bool) {
```

Comment:
I think `centerToolbarItems` should default to `false` here. No point in forcing users to specify that value when most will want the default of not centering.

Reply:
Oh, did you notice it's not part of the public API? Since this is only internal, I got rid of the default to highlight that the parameter exists at all. More like a humble reminder for future changes than something users will see. Still want me to change this?

Reply:
I missed that. Ignore my comment.